frontend/src/core/models/task/task-priority.ts

```diff
@@ -0,0 +1,6 @@
+export interface TaskPriority {
```

Comment:
I think, it would be better to use enum here. We do not need 'createdAt', 'updatedAt' for priorities